### PR TITLE
added shared_s3 bucket nameing format

### PR DESCRIPTION
**Pull request by ndcampbell**

### CODE REVIEW

**Comment by ndcampbell** [comment]
CI is failing cause of prospector issue

File: src/gogoutils/formats.py [hunk 1]
```diff
@@ -40,6 +40,7 @@
     'iam_user': '{project}_{repo}',
     'jenkins_job_name': '{project}_{repo}',
     's3_app_bucket': '{project}-{repo}',
+    'shared_s3_app_bucket': 'common-{project}-{repo}',
```
Comment (sijis) [inline src/gogoutils/formats.py]: Wouldn't a better default be `common-{project}`?

Comment (ndcampbell) [inline src/gogoutils/formats.py]: Yeah, that might make more sense. I dont really have a preference since it is all overridden by local formats

Comment (sijis) [inline src/gogoutils/formats.py]: True but the default should be sensible.

**Comment by sijis** [comment]
You'll need to fix the test case:

```
========================================================================================= FAILURES ==========================================================================================
___________________________________________________________________________________ test_s3_bucket_format ___________________________________________________________________________________

    def test_s3_bucket_format():
        """Validates generated s3 bucket name"""
        for project in PROJECTS:
    
            g = Generator(
                PROJECTS[project]['project'],
                PROJECTS[project]['repo'],
                PROJECTS[project]['env'],
            )
    
            bucket = '{1}-{0}'.format(
                PROJECTS[project]['repo'],
                PROJECTS[project]['project'],
            )
    
            shared_bucket = 'common-{1}-{0}'.format(
                PROJECTS[project]['repo'],
                PROJECTS[project]['project'],
            )
    
            assert bucket == g.s3_app_bucket()
>           assert shared_bucket == g.shared_s3_app_bucket()
E           assert 'common-gogoair-test' == 'common-gogoair'
E             - common-gogoair-test
E             ?               -----
E             + common-gogoair

tests/test_generator.py:297: AssertionError
============================================================================ 1 failed, 17 passed in 0.16 seconds ============================================================================
```

**Pull request merged**